cli/pcluster/cfnconfig.py

```diff
@@ -347,19 +336,89 @@ def __init_vpc_parameters(self):
             try:
                 __temp__ = self.__config.get(vpc_section, key)
                 if not __temp__:
-                    print("ERROR: %s defined but not set in [%s] section" % (key, vpc_section))
-                    sys.exit(1)
+                    self.__fail("%s defined but not set in [%s] section" % (key, vpc_section))
                 if vpc_options.get(key)[1] is not None:
                     self.__validate_resource(vpc_options.get(key)[1], __temp__)
                 self.parameters[vpc_options.get(key)[0]] = __temp__
             except configparser.NoOptionError:
                 pass
             except configparser.NoSectionError:
-                print(
-                    "ERROR: VPC section [%s] used in [%s] section is not defined"
-                    % (vpc_section, self.__cluster_section)
+                self.__fail(
+                    "VPC section [%s] used in [%s] section is not defined" % (vpc_section, self.__cluster_section)
                 )
-                sys.exit(1)
+
+    def __check_account_capacity(self):
+        """Try to launch the requested number of instances to verify Account limits."""
+        test_ami_id = self.__get_latest_alinux_ami_id()
```

Comment:
Why do you need this ami?

Reply:
Because the `run_instances` command requires an AMI id, so it is a way to found an ami id for the test.